src/openrpc/lifecycle_management.json

```diff
@@ -64,7 +64,7 @@
         },
         {
             "name": "session",
-            "summary": "Used by a Lifecycle provider to tell Firebolt that an app session has been started or updated. Firebolt can use this information to have context about the app when it makes API requests",
+            "summary": "Called before an app is launched in order to set up the lifecycle session. The session should be given an intent that will be passed to the application. This will behave differently depending on the current state of the application. If the application is already loaded but not active then a new active session is started. The intent is given to the application via the onNavigateTo event. If the app is not loaded, then it is considered a 'cold start'. A new loaded session will be created and the intent will be stored for the app to access via Parameters.initialization. Before a session is created for the application some checks may take place. For instance, if a user grant is required for the application then the user grant will be resolved before the session is created. In this case, this API will return immediately with a state='pending'. The created session will then be sent as an event in the LifecycleManagement.onSessionCreated event. If the pre-session checks fail, such as a user deferring a grant, then the LifecycleManagement.onSessionCanceled event will be emitted.",
```

Comment:
What is the workflow for the app to register for these events ? I mean when do the app register for these events ?

Reply:
Can you also confirm  that the GrantPolicy changes with evaluateAt is updated in Firebolt-open-rpc ?

I found a scenario where in 'cold start' mode with launch state active, both loaded_session_id & active_session_id will be created anew.  

Is there any GrantPolicy requirement to evaluate user grant at a new ActiveSession OR LoadedSession ?

Reply:
Apps should register for these events when they start up

Reply:
I will add evaluateAt to the GrantPolicy spec